diff --git a/android/src/main/java/org/reactnavigation/MaterialSymbolModule.kt b/android/src/main/java/org/reactnavigation/MaterialSymbolModule.kt
index 1f21c138d0d42e31981b571adbaddb9b637ebfc5..1eb28e1d2b9ed196c1f90f8419ce7786adef2950 100644
--- a/android/src/main/java/org/reactnavigation/MaterialSymbolModule.kt
+++ b/android/src/main/java/org/reactnavigation/MaterialSymbolModule.kt
@@ -46,7 +46,7 @@ class MaterialSymbolModule(reactContext: ReactApplicationContext) :
     }
 
     val resolvedColor = ColorPropConverter.getColor(
-      colorValue, currentActivity ?: reactApplicationContext
+      colorValue, reactApplicationContext.currentActivity ?: reactApplicationContext
     ) ?: throw IllegalArgumentException("Could not resolve color")
 
     val density = reactApplicationContext.resources.displayMetrics.density
