diff --git a/lib/module/views/BottomTabViewNativeImpl.js b/lib/module/views/BottomTabViewNativeImpl.js
index 8022f2f..dfbfa43 100644
--- a/lib/module/views/BottomTabViewNativeImpl.js
+++ b/lib/module/views/BottomTabViewNativeImpl.js
@@ -407,12 +407,14 @@ export function BottomTabView(props) {
             android: {
               icon: icon?.android ?? icon?.shared,
               selectedIcon: selectedIcon?.android ?? selectedIcon?.shared,
               standardAppearance: {
                 tabBarBackgroundColor: tabBarBackgroundColor ?? backgroundColor,
-                tabBarItemRippleColor: currentOptions?.tabBarRippleColor,
+                tabBarItemRippleColor: currentOptions?.tabBarRippleColor ?? Color(activeTintColor)?.alpha(0.12).string(),
                 tabBarItemLabelVisibilityMode: currentOptions?.tabBarLabelVisibilityMode,
                 normal: normalTabItemAppearance,
                 selected: selectedTabItemAppearance,
+                focused: normalTabItemAppearance,
+                disabled: normalTabItemAppearance,
                 tabBarItemActiveIndicatorColor: activeIndicatorColor,
                 tabBarItemActiveIndicatorEnabled: currentOptions?.tabBarActiveIndicatorEnabled,
                 tabBarItemTitleFontFamily: fontFamily,
diff --git a/src/views/BottomTabViewNativeImpl.tsx b/src/views/BottomTabViewNativeImpl.tsx
index a5cf142..25c292c 100644
--- a/src/views/BottomTabViewNativeImpl.tsx
+++ b/src/views/BottomTabViewNativeImpl.tsx
@@ -601,11 +601,15 @@ export function BottomTabView(props: Props) {
                 standardAppearance: {
                   tabBarBackgroundColor:
                     tabBarBackgroundColor ?? backgroundColor,
-                  tabBarItemRippleColor: currentOptions?.tabBarRippleColor,
+                  tabBarItemRippleColor:
+                    currentOptions?.tabBarRippleColor ??
+                    Color(activeTintColor)?.alpha(0.12).string(),
                   tabBarItemLabelVisibilityMode:
                     currentOptions?.tabBarLabelVisibilityMode,
                   normal: normalTabItemAppearance,
                   selected: selectedTabItemAppearance,
+                  focused: normalTabItemAppearance,
+                  disabled: normalTabItemAppearance,
                   tabBarItemActiveIndicatorColor: activeIndicatorColor,
                   tabBarItemActiveIndicatorEnabled:
                     currentOptions?.tabBarActiveIndicatorEnabled,
