diff --git a/node_modules/@tamagui/config/.bun-tag-f639c8dbc8ce2f4b b/.bun-tag-f639c8dbc8ce2f4b
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/dist/esm/v5.native.js b/dist/esm/v5.native.js
index 713e163997f14237819c6f351274dbad4b2ce3d8..d51daff91989bd8054e7ce152353a36f8ba845ea 100644
--- a/dist/esm/v5.native.js
+++ b/dist/esm/v5.native.js
@@ -1,3 +1,2 @@
-export * from "@tamagui/themes/v5";
-export * from "./v5-base.native.js";
-//# sourceMappingURL=v5.native.js.map
+// explicit re-exports to avoid ESM export * lazy getter issues on Hermes V1
+export { adjustPalette, adjustPalettes, breakpoints, createSystemFont, createThemes, createV5Theme, defaultChildrenThemes, defaultConfig, defaultDarkPalette, defaultLightPalette, fonts, hslToString, interpolateColor, media, mediaQueryDefaultActive, opacify, parseHSL, selectionStyles, settings, shorthands, tokens } from "./v5-base.native.js";
