diff --git a/index.js b/index.js
index 23384ea16bb848616886f356d8d69f94325535bf..3c5e8f4e3c41295115f07b47fa310b1cd67d9ea6 100644
--- a/index.js
+++ b/index.js
@@ -126,6 +126,13 @@ class Extractor {
 
     if (symlink) {
       const link = await getStream(readStream)
+      const resolvedLink = path.resolve(destDir, link)
+      const relativeLink = path.relative(this.opts.dir, resolvedLink)
+
+      if (relativeLink === '..' || relativeLink.startsWith(`..${path.sep}`) || path.isAbsolute(relativeLink)) {
+        throw new Error(`Out of bound symlink target "${link}" found while processing file ${entry.fileName}`)
+      }
+
       debug('creating symlink', link, dest)
       await fs.symlink(link, dest)
     } else {
